### ISSUE TITLE
SSL error when running rake db_connector:three11

### ISSUE THREAD

**Issue by LKWLaLa** [chrishenry/deltanyc-heatseek#62]
```
**************
BX

**************
MN

**************
BK

rake aborted!
OpenSSL::SSL::SSLError: SSL_connect SYSCALL returned=5 errno=0 state=unknown state
/usr/local/bundle/gems/httparty-0.13.7/lib/httparty/request.rb:117:in `perform'
/usr/local/bundle/gems/httparty-0.13.7/lib/httparty.rb:545:in `perform_request'
/usr/local/bundle/gems/httparty-0.13.7/lib/httparty.rb:476:in `get'
/usr/local/bundle/gems/httparty-0.13.7/lib/httparty.rb:583:in `get'
/angular-rails-app/lib/tasks/db_connector.rake:28:in `nyc_geocode'
/angular-rails-app/lib/tasks/db_connector.rake:394:in `block (3 levels) in <top (required)>'
/angular-rails-app/lib/tasks/db_connector.rake:382:in `each'
/angular-rails-app/lib/tasks/db_connector.rake:382:in `block (2 levels) in <top (required)>'
/usr/local/bundle/gems/rake-12.0.0/exe/rake:27:in `<top (required)>'
Tasks: TOP => db_connector:three11
```

Was previously getting a Net::OpenTimeout error : 
```**************
QN

**************
BX

**************
BK

**************
MN

rake aborted!
Net::OpenTimeout: Net::OpenTimeout
/usr/local/bundle/gems/httparty-0.13.7/lib/httparty/request.rb:117:in `perform'
/usr/local/bundle/gems/httparty-0.13.7/lib/httparty.rb:545:in `perform_request'
/usr/local/bundle/gems/httparty-0.13.7/lib/httparty.rb:476:in `get'
/usr/local/bundle/gems/httparty-0.13.7/lib/httparty.rb:583:in `get'
/angular-rails-app/lib/tasks/db_connector.rake:28:in `nyc_geocode'
/angular-rails-app/lib/tasks/db_connector.rake:394:in `block (3 levels) in <top (required)>'
/angular-rails-app/lib/tasks/db_connector.rake:382:in `each'
/angular-rails-app/lib/tasks/db_connector.rake:382:in `block (2 levels) in <top (required)>'
/usr/local/bundle/gems/rake-12.0.0/exe/rake:27:in `<top (required)>'
Tasks: TOP => db_connector:three11
```

**Comment by LKWLaLa** [comment]
We should handle the SSL error when calling that function, and move on if it occurs.

**Comment by rdlester** [comment]
I am 99% certain this is fixed by the linked PR ( #76 )
